src/main/steps/common/form/form.njk

```diff
@@ -93,7 +93,7 @@
             name: "onlycontinue",
             value: true,
             attributes: {
-              id: 'main-form-save-as-draft'
+              id: 'main-form-only-continue'
```

Comment:
Is the id used anywhere in the code? Is the change tested and working without breaking any functionality.

Reply:
Not used anywhere else in application, no breaking functionality observed during testing